formula/src/main/java/com/instacart/formula/FormulaPlugins.kt

```diff
@@ -18,4 +18,12 @@ object FormulaPlugins {
             else -> ListInspector(listOf(global, local))
         }
     }
+
+    fun onDuplicateChildKey(
```

Comment:
Do we want to generalize this to support other formula errors in the future? So something like:
```
onError(error: FormulaError)
```

where `FormulaError` is a sealed interface

```
sealed interface FormulaError {
    data class DuplicateKeyError(
        val parentFormulaType: KClass<*>,
        val childFormulaType: KClass<*>,
        val key: Any,
    ) : FormulaError
}
```

Reply:
Maybe in the future, but for now doesn't seem necessary

Reply:
Given it's pretty easy to migrate to a different design, I prefer waiting for more than one use case before designing a generic thing.